t/ui/23-audit-log.t

```diff
@@ -64,7 +68,7 @@ subtest 'audit log entries' => sub {
     my $search = $driver->find_element('#audit_log_table_filter input.form-control');
     ok($search, 'search box found');
 
-    check_data_table_entries $table, 4, 'four rows without filter';
+    check_data_table_entries $table, 5, 'four rows without filter';
```

Comment:
```suggestion
    check_data_table_entries $table, 5, 'five rows without filter';
```

?

Although the test description isn't really good. Why should we care that we have exactly four?